### DISCUSSION TITLE
Porque no me aparece el driver Jack en apps

### DISCUSSION THREAD

**Discussion by leaxdrom** [jackaudio/jack2#850] [Q&A]
Que tal , ya instale jack 2 en mac catalina pero no me aparece el driver en apps para rutear la señal, que debo hacer? gracias

**Comment by falkTX** [comment]
Use english only please.

Applications must support JACK natively in order to use it, that is to say, regular applications are not supported and will not appear in the jack graph

**Reply by leaxdrom** [reply]
HELLO EXCUSE ME
IN GOOGLE I HAVE A TRANSLATOR OF THE PAGE TO SPANISH.
I GET THIS MESSAGE IN JACK'S WINDOW...
12:30:19.631 Statistics reset.
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
12:30:32.992 JACK is starting...
12:30:32.992 /usr/local/bin/jackd -dcoreaudio -dAppleHDAEngineInput:1B,0,1,0:1 -r48000 -p128
Cannot connect to server socket err = No such file or directory
Cannot connect to server request channel
jack server is not running or cannot be started
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
JackShmReadWritePtr::~JackShmReadWritePtr - Init not done for -1, skipping unlock
12:30:33.007 JACK was started with PID=2304.
jackdmp 1.9.20
Copyright 2001-2005 Paul Davis and others.
Copyright 2004-2016 Grame.
Copyright 2016-2021 Filipe Coelho.
jackdmp comes with ABSOLUTELY NO WARRANTY
This is free software, and you are welcome to redistribute it
under certain conditions; see the file COPYING for details
JACK server starting in realtime mode with priority 10
self-connect-mode is "Don't restrict self connect requests"
Input channel = 0 ==> JACK input port = 0
Input channel = 1 ==> JACK input port = 1
12:30:37.066 JACK connection change.
12:30:37.067 Server configuration saved to "/Users/usermac/.jackdrc".
12:30:37.068 Statistics reset.
12:30:37.074 Client activated.
12:30:37.076 Patchbay deactivated.
CoreAudio driver is running...
12:30:37.081 JACK connection graph change.
I HAVE TRIED TO CONNECT IT IN AUDACITY AND IN REAPER,THANK YOU.
